subsystem/kubeapi_test.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	conditionsv1 "github.com/openshift/custom-resource-status/conditions/v1"
```

Comment:
Why is that changed? goimports automatically changed it for you?

Reply:
yup.
will get back to line 17 in the next update. thanks!

Reply:
Done.